subgraphs/inverse-finance/src/common/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { SECONDS_PER_HOUR, SECONDS_PER_DAY } from "./constants";
+
+// Converts snake case to kebab case and appends a hyphen.
+// (e.g. "TRADING_FEE" to "trading-fee-"), mainly used to create entity IDs
+export function enumToPrefix(snake: string): string {
+  return snake.replace("_", "-") + "-";
+}
+
+// Prefix an ID with a enum string in order to differentiate IDs
+// e.g. combine TRADING_FEE and 0x1234 into TRADING-FEE-0x1234
+export function prefixID(ID: string, enumString1: string, enumString2: string | null = null): string {
```

Comment:
This is helpful. Would you also update the version in `_reference_` subgraph?

Reply:
For this version of `prefixID`, I swapped the position for ID and enumString argument to make it less confusing/error prone in use. I hope that's OK. 
Also, do you prefer replacing the existing prefixID function in _reference_ or creating another function with a different name?

Reply:
ah, yes, that won't be good as it will mix things up. Let's just leave it as is then.